### PR TITLE
Closes #714 Add metatag field to all content types

### LINKED ISSUE #714: Add metatag field to each of the content types
Drupal 7 metatag module does not require a field be added for metatags to be updated or changed, but in Drupal 8/9 you must add a metatag field to each content type that you want to allow overrides on.

We should add the field to:
Page
Event
Person
News

Hopefully we can add the field in the optional config folder as not to make the SEO module a dependency.

### PR DESCRIPTION
**Pull request by smayden**
This PR add the field_az_metatag field to the Events, News, Page, and Person content types

## Description
The metatag module in D9 requires that a field is created and added to the content type. This PR creates field_az_metatag and updates the four content types.

@trackleft helped sort out file locations, so that metatag module is not a dependency for these content types.

The default meta description token has been updated to [node:field_az_summary] rather than [node:summary] — we may want to add field_az_summary to person eventually since the other three content types have it.

## Related Issue
#714 

## How Has This Been Tested?
Tested locally and in Probo

Steps to test:
1. Add content (Page, News, Event, Person)
2. Verify that you see the metatag dropdown on the right side of the screen
3. Open metatag and verify that you see the token [node:field_az_summary]
4. Go to summary tab on right hand side. Add text in the summary field
5. Inspect the content and verify that the text from summary was included as the metadescription
6. Edit the content again, go back to metatag and enter a custom metadescription instead of the token
7. Save
8. Inspect the content and verify that the custom metadescription shows up  

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_seo/config/install/field.storage.node.field_az_metatag.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - metatag
```
Comment (trackleft) [inline modules/custom/az_seo/config/install/field.storage.node.field_az_metatag.yml:5]:
```suggestion
    - az_seo
```

**Comment by trackleft** [comment]
Adding form overrides to the quickstart folder of az_seo, though a better option might be an install hook for config overrides on quickstart provided config in az_seo.

**Comment by smayden** [comment]
This ticket should be ready to review: https://a6bc5100-d2c7-4365-8633-2d8b627d9f65.probo.build/

**Comment by trackleft** [comment]
To anyone reviewing, the downside to this approach is that 

1. We are maintaining the node edit form in two places... Within the node module (example az_person) and within the quickstart folder of az_seo.
2. Any future modules that require changing those  forms will have to be done in a different way because we are already using the override functionality for this.

Advantages az_seo module is still optional.

File: modules/custom/az_seo/config/quickstart/core.entity_form_display.node.az_event.default.yml [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+      format_settings:
+        id: ''
+        classes: ''
+        description: 'For use in feeds and content listings.'
+        open: true
+        required_fields: true
+        weight: 0
+      label: 'Summary fields'
+_core:
+  default_config_hash: '-PrT7Z_cg-V1sqCMF0QUVxk6fP3Z-uUVZwqj1EnG4_0'
```
Comment (joeparsons) [inline modules/custom/az_seo/config/quickstart/core.entity_form_display.node.az_event.default.yml:47]: I think we want to remove these lines.
```suggestion
```

File: modules/custom/az_seo/config/quickstart/core.entity_form_display.node.az_news.default.yml [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+      format_settings:
+        id: ''
+        classes: ''
+        description: ''
+        open: false
+        required_fields: true
+        weight: 0
+      label: 'Extra Fields'
+_core:
+  default_config_hash: 4mNfedvNkrf-BeMP6PKElcTrMR0JpJ07E8SHTFwwUBo
```
Comment (joeparsons) [inline modules/custom/az_seo/config/quickstart/core.entity_form_display.node.az_news.default.yml:64]:
```suggestion
```

Comment (tadean) [inline modules/custom/az_seo/config/quickstart/core.entity_form_display.node.az_news.default.yml]:
```suggestion
```

**Review by trackleft — APPROVED** [review]

**Comment by smayden** [comment]
We discussed in the Wednesday PR review that we were ok with the having the form display override in the az_seo module, but will look at different approaches in the future if we need more complicated overrides.

This just needs one more checkmark.

**Review by tadean — APPROVED** [review]

**Review by joeparsons — APPROVED** [review]

**Pull request merged**